app/src/main/kotlin/view/utils/Neo4jLoginDialog.kt

```diff
@@ -0,0 +1,161 @@
+package view.utils
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.HorizontalAlignmentLine
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import kotlinx.coroutines.delay
+import viewmodel.MainScreenViewModel
+
+@Composable
+fun <D> Neo4jLoginDialog(
+    mainScreenViewModel: MainScreenViewModel<D>,
```

Comment:
Why do you need MSVM?

Reply:
I looked over my decisions and I agree with you